frontend/src/metabase/lib/urls/browse.ts

```diff
@@ -0,0 +1,28 @@
+import slugg from "slugg";
+
+import { SAVED_QUESTIONS_VIRTUAL_DB_ID } from "metabase/lib/saved-questions";
+import { Database, Table } from "metabase-types/api";
+
+import { appendSlug } from "./utils";
+
+export function browseDatabase(database: Database) {
+  const name =
+    database.id === SAVED_QUESTIONS_VIRTUAL_DB_ID
+      ? "Saved Questions"
+      : database.name;
+
+  return appendSlug(`/browse/${database.id}`, slugg(name));
+}
+
+type TableWithDatabaseAndSchemaInfo = Table & {
```

Comment:
`db` and `schema_name` are returned by backend when `included` in request params. WDYT about adding these properties to `Table`, since they are actually properties of this entity returned by the backend? In this case we would have to make them optional, I guess.

Reply:
Thanks, @ranquild! Fixed: added `db` and `schema_name` as optional properties on the `Table` interface